### PR TITLE
Symfony 3.4 support

### PR DESCRIPTION
**Pull request by TomasLudvik**
**Description, reason for the PR:** upgrade of the main repository to Symfony 3.4

**New feature:** No

**BC breaks:** No

**Fixes issues:** None

**Standards and tests pass:** Yes

**License:** MIT

### CODE REVIEW

File: src/Resources/config/services.yml [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
+    public: false
 
-  shopsys.form_types.yes_no:
-    class: Shopsys\FormTypesBundle\YesNoType
-    tags:
-      - { name: form.type }
+  Shopsys\FormTypesBundle\:
+    resource: '../../*'
+
+  Shopsys\FormTypesBundle\Domain\DomainIdsProviderInterface:
```
Comment (vitek-rostislav) [inline src/Resources/config/services.yml]: this should be in services.yml in shopsys framework

**Review by vitek-rostislav — CHANGES_REQUESTED** [review]
Hi, CR is done :slightly_smiling_face:

File: src/Resources/config/services.yml (hunk 1 above)
Comment (TomasLudvik) [reply src/Resources/config/services.yml]: fixed

**Comment by vitek-rostislav** [comment]
Hi, seems okay, you can merge :+1:

**Pull request merged**